### DISCUSSION TITLE
Feature: Double-click to close tab

### DISCUSSION THREAD

**Discussion by ablazewicz** [zen-browser/desktop#2090] [Ideas]
Zen Browser is great, but I miss one important feature I know from Vivaldi, namely the ability to close a tab with a double click. This would be very convenient, especially on laptops.

**Comment by MannixMD** [comment]
right now you can enable browser.tabs.closeTabByDblclick in about:config :)

**Reply by ablazewicz** [reply]
Cool! That was the point. :) Thank you!
